widgetastic_manageiq.py

```diff
@@ -1671,3 +1675,18 @@ def fill(self, values):
             if field.fill({'attribute': key, 'value': value}):
                 changed = True
         return changed
+
+
+class FileInput(BaseFileInput):
+    """ represents enhanced FileInput control.
+    Accepts a string. If the string is a file, then it is put in the input. Otherwise a temporary
+    file is generated and that one is fed to the file input.
+    """
+    def fill(self, value):
+        if not os.path.isfile(value):
+            f = NamedTemporaryFile()
```

Comment:
this introduces a requirement for out of band ressource and file management, we should avoid something like that

while this is merge-able as it adds functionality, we should clearly mark this as technical debt needing a better resource management exposed from widgettastic or our wrappers